fili-core/src/main/java/com/yahoo/bard/webservice/data/PartialDataHandler.java

```diff
@@ -3,14 +3,14 @@
 package com.yahoo.bard.webservice.data;
 
 import static com.yahoo.bard.webservice.config.BardFeatureFlag.PERMISSIVE_COLUMN_AVAILABILITY;
-import static com.yahoo.bard.webservice.util.TableUtils.getColumnNames;
```

Comment:
If the IDE gets confused, that's a good reason not to be a static import, but in general, static imports reduce verbosity and improve code clarity.